src/services/serverApi.ts

```diff
@@ -83,34 +83,35 @@ export const {
 } = serverApi;
 
 export const useGetMembers = () => {
-  const { data, isLoading, isFetching, error } = serverApi.useGetMembersQuery()
-  const membersWithRole = data?.members?.filter(
-    (member: MemberType) =>
-      member?.isMember === true && member?.first_name && !member.roles.archived
+  const { data, isLoading, isFetching, error } = serverApi.useGetAllUsersQuery();
+
+  const usersWithMemberRole = data?.users?.filter(
+    (member: UserType) =>
+      member?.roles.member === true && member?.first_name && !member.roles.archived
   );
-  const sortedUsers = membersWithRole?.sort((a,b) => a.first_name > b.first_name ? 1 : -1) 
 
+  const sortedMembers = usersWithMemberRole?.sort((a, b) => a.first_name > b.first_name ? 1 : -1)
   return {
-    data: sortedUsers,
+    data: sortedMembers,
     isLoading,
     isFetching,
     error
   }
 }
 
 export const useGetUsers = () => {
-  const { data, isLoading, isFetching, error } = serverApi.useGetMembersQuery()
-  const membersWithRole = data?.members?.filter(
-    (member: MemberType) =>
-      member?.isMember === false &&
-      member?.first_name &&
-      !member.roles.archived &&
-      member.roles.in_discord
+  const { data, isLoading, isFetching, error } = serverApi.useGetAllUsersQuery()
+  const usersWithoutMemberRole = data?.users?.filter(
+    (nonMember: UserType) =>
+      !nonMember?.roles.member &&
+      nonMember?.first_name &&
+      !nonMember.roles.archived &&
+      nonMember.roles.in_discord
   );
-  const sortedUsers = membersWithRole?.sort((a,b) => a.first_name > b.first_name ? 1 : -1) 
+  const sortedNonMembers = usersWithoutMemberRole?.sort((a, b) => a.first_name > b.first_name ? 1 : -1)
```

Comment:
- Why are we performing this sort?

Reply:
To show the Non-members in an Alphabetical Order w.r.t their first name.

Reply:
- please add a comment stating the same.